scripts/install_hab_pkg.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+export HAB_NONINTERACTIVE=true
+export HAB_NOCOLORING=true
+export HAB_LICENSE="accept-no-persist"
+
+ident="${1:-ident required}"
```

Comment:
Should we use `?` here instead? 
```suggestion
ident="${1?ident required}"
```

Reply:
my bad! thanks